AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-participant-list/booking-participant-list.component.ts

```diff
@@ -40,6 +40,7 @@ export class BookingParticipantListComponent {
         const panelMembersAndWingers = this.participants.filter(participant =>
             ['Panel Member', 'Winger'].includes(participant.HearingRoleName)
         );
+        const staffMember = this.participants.filter(participant => participant.HearingRoleName === 'Staff Member');
```

Comment:
could we derive the magic string from the CaseRoles enum here for staff member?

Reply:
HearingRoles and CaseRoles enums are diff in this case, this could bring a confusion into the logic

Reply:
ok makes sense

Reply:
case roles enum will be extended as part of a continuation of this ticket
P.S.: you are right in not using "magic strings" for thisis the right thign to do